core/src/main/java/eu/fasten/core/maven/resolution/ResolverConfig.java

```diff
@@ -48,11 +48,34 @@ public ResolverConfig at(long timestamp) {
         return this;
     }
 
-    public ResolverConfig depth(ResolverDepth depth) {
+    public ResolverConfig includeTransitiveDeps() {
+        this.depth = Integer.MAX_VALUE;
+        return this;
+    }
+
+    public ResolverConfig excludeTransitiveDeps() {
```

Comment:
If the depth is 1, would it be better to name it `includeDirectDeps`?

Reply:
That would be misleading as it is always the case... direct dependencies are also included when you resolve the transitive ones. The only difference in configuration is the extent at which transitive dependencies are included, so I have named the three relevant methods accordingly.